unime/src-tauri/tauri.conf.json

```diff
@@ -58,11 +58,12 @@
   "plugins": {
     "deep-link": {
       "mobile": [
+        { "scheme": [ "unime" ] },
```

Comment:
In order for this to work on iOS, make sure to add the following to `unime/src-tauri/gen-static/apple/unime_iOS/Info.plist` in L20:

```
<dict>
	<key>CFBundleTypeRole</key>
	<string>Viewer</string>
	<key>CFBundleURLName</key>
	<string>com.impierce.identity-wallet</string>
	<key>CFBundleURLSchemes</key>
	<array>
		<string>unime</string>
	</array>
</dict>
```